cm-mlops/script/get-ml-model-resnet50-tvm/_cm.json

```diff
@@ -117,13 +117,19 @@
           "names": [
             "original-model"
           ],
-          "tags": "get,ml-model,raw,resnet50,_tf"
+          "tags": "get,ml-model,raw,resnet50,_tflite"
```

Comment:
Should this be another variation tflite as we also have a tensorflow model?

Reply:
I agree, I fixed it